test/views/message_view_test.js

```diff
@@ -1,17 +1,19 @@
 describe('MessageView', function() {
   var convo, message;
 
-  before(function() {
+  before(async (done) => {
+    await clearDatabase();
     convo = new Whisper.Conversation({id: 'foo'});
     message = convo.messageCollection.add({
       conversationId: convo.id,
       body: 'hello world',
       type: 'outgoing',
       source: '+14158675309',
-      received_at: Date.now()
+      received_at: Date.now(),
```

Comment:
Fun fact - `tslint`'s default recommended setup enforces trailing commas like this. :0)